src/widgets/trOverview.test.js

```diff
@@ -251,7 +251,7 @@ describe('TR overview widget', () => {
       numReports: '2',
       numSessions: '4',
       recipientPercentage: '60.00%',
-      sumDuration: '4',
+      sumDuration: '4.0',
```

Comment:
should you make this something like 4.15 and assert its like 4.2

Reply:
We don't collect numbers like that (.15). I will make it a 4.5.